carbonmark-api/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@klimadao/carbonmark-api",
-  "version": "4.0.1",
+  "version": "4.1.1",
```

Comment:
This should be 4.1.0 no?

Reply:
Actually 4.0.2 since it's not a "backwards compatible feature"

Reply:
fixed